index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started overview
-summary: "GraphWalker is a open source Model-based testing tool for test automation. It's designed to make it easy to design your tests using graphs."
+summary: "GraphWalker is an open source Model-based testing tool for test automation. 
```

Comment:
This is not really a change, is it? You added some new lines, but the mark down rendering is still the same, right?
If so, I suggest not to make a change here.